authorizer/urm.go

```diff
@@ -6,19 +6,19 @@ import (
 	"github.com/influxdata/influxdb/v2"
 )
 
-type OrganizationService interface {
+type OrgIDResolver interface {
```

Comment:
I'm not objecting to the name, but throwing out `OrgIDFinder` as an alternative, in case you like it and the verb is **Find**ResourceOrganizationID.

Reply:
Good shout. I actually rename this again in two PRs time to just `Resolver` (in the Telegraf PR). But I will rename it Finder in that PR as it finds both org IDs and name 👍